src/Routes.tsx

```diff
@@ -12,12 +12,15 @@ const PrivacyPolicy = React.lazy(() => import("./routes/PrivacyPolicy"));
 export function Routes() {
   return (
     <React.Suspense fallback={null}>
-      <Route exact path="/" component={Home} />
-      <Route exact path="/tos" component={TermsOfService} />
-      <Route exact path="/privacypolicy" component={PrivacyPolicy} />
-      <Route exact path="/search/:postalCode" component={Search} />
-      <Route exact path="/login" component={Login} />
-      <PrivateRoute path="/admin" render={() => <AdminRoutes />} />
+      <Switch>
+        <Route exact path="/" component={Home} />
+        <Route exact path="/tos" component={TermsOfService} />
+        <Route exact path="/privacypolicy" component={PrivacyPolicy} />
+        <Route exact path="/search/:postalCode" component={Search} />
+        <Route exact path="/login" component={Login} />
+        <PrivateRoute path="/admin" render={() => <AdminRoutes />} />
+        <Redirect exact from="*" to="/" />
```

Comment:
This works, but I think it would be more direct to just write `<Redirect to="/" />` Right now this reads as "match exactly the route pattern that matches any route pattern".